extensions/data-plane-selector/selector-api/src/main/java/org/eclipse/dataspaceconnector/dataplane/selector/api/DataplaneSelectorApiController.java

```diff
@@ -28,6 +29,7 @@
 @Consumes({ MediaType.APPLICATION_JSON })
 @Produces({ MediaType.APPLICATION_JSON })
 @Path("/instances")
+@Tag(name = "Dataplane Selector")
```

Comment:
We agreed on putting the swagger annotations on the `*Api` interface, not on the `*ApiController` implementation. In this case we need to extract the interface as well